spec/controllers/api/sequences/sequences_create_spec.rb

```diff
@@ -21,5 +21,20 @@
       post :create, input
       expect(response.status).to eq(200)
     end
+
+    it 'handles invalid params' do
+      # Needed to test the `else` branch of mutate() somewhere
+      sign_in user
+      input = { steps: [{ message_type: 'move_rel',
+                          command: { action: 'MOVE RELATIVE',
+                                     x: 1,
+                                     y: 2,
+                                     z: 3,
+                                     speed: 100,
+                                     delay: 0} }] }
+      post :create, input
+      expect(response.status).to eq(422)
+      expect(json[:name]).to eq("Name is required")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.